app/css/openmrs-owa-orderentry.scss

```diff
@@ -7,105 +7,130 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 
-
 body {
-    font-family: "OpenSans", Arial, sans-serif;
-    -webkit-font-smoothing: subpixel-antialiased;
-    max-width: 1000px;
-    margin: 10px auto;
-    background: #eeeeee;
-    color: #363463;
-    font-size: 16px;
+  font-family: 'OpenSans', Arial, sans-serif;
+  -webkit-font-smoothing: subpixel-antialiased;
+  max-width: 1000px;
+  margin: 10px auto;
+  background: #eeeeee;
+  color: #363463;
+  font-size: 16px;
 }
 
 .body-wrapper {
-    padding-top: 15px;
-    padding-left: 10px;
-    padding-right: 10px;
-    background-color: white;
-    -moz-border-radius: 5px;
-    -webkit-border-radius: 5px;
-    -o-border-radius: 5px;
-    -ms-border-radius: 5px;
-    -khtml-border-radius: 5px;
-    border-radius: 0px 0px 5px 5px;
-    min-height: 500px;
-    justify-content: center;
+  padding-top: 15px;
+  padding-left: 10px;
+  padding-right: 10px;
+  background-color: white;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: 5px;
+  -o-border-radius: 5px;
+  -ms-border-radius: 5px;
+  -khtml-border-radius: 5px;
+  border-radius: 0px 0px 5px 5px;
+  min-height: 500px;
+  justify-content: center;
 }
 
 .app-title {
-    display: block;
-    float: right;
-    color: #eeeeee;
-    font-size: 32px;
-    margin: 4px;
-    height: 100%;
-    padding-top: 12px;
-    padding-right: 5px;
+  display: block;
+  float: right;
+  color: #eeeeee;
+  font-size: 32px;
+  margin: 4px;
+  height: 100%;
+  padding-top: 12px;
+  padding-right: 5px;
 }
 
 .instructions {
-    align-self: center;
+  align-self: center;
 }
 
 .show {
-    display: block;
+  display: block;
 }
 
 .icon-color {
-    color: #007fff;
+  color: #007fff;
 }
 
 .draft-spacing {
-    padding-top: 1em;
-    padding-bottom: 2em;
-    background-color: #fff;
+  padding-top: 1em;
+  padding-bottom: 2em;
+  background-color: #fff;
 }
 
 .hide {
-    display: none;
+  display: none;
 }
 
 .text-align-center {
-    text-align: center
+  text-align: center;
 }
 
 .fs-14-px {
-    font-size: 14px
+  font-size: 14px;
 }
 
 .w-18-percent {
-    width: 18%
+  width: 18%;
 }
 
 .w-10-percent {
-    width: 10%
+  width: 10%;
 }
 
 .w-44-percent {
-    width: 44%
+  width: 44%;
 }
 
 .mw-958-px {
-    min-width: 958px;
+  min-width: 958px;
 }
 
 .w-155-px {
-    width: 155px;
+  width: 155px;
 }
 
 .w-81-px {
-    width: 81px;
+  width: 81px;
 }
 
 .w-120-px {
-    width: 120px;
+  width: 120px;
 }
 
 .pl-39-px {
-    padding-left: 39px;
+  padding-left: 39px;
 }
 
 .action-padding {
-    padding: 10px 10px 10px 25px;
+  padding: 10px 10px 10px 25px;
+}
+
+.rotate90 {
+  transform: rotate(90deg);
+}
+.rotate180::before {
+  transform: rotate(180deg);
+}
+
+.accordion-two.content {
+  padding: 0px 5px;
+  overflow-y: scroll;
+  max-height: 0px;
+  transition: all 0.5s ease;
+}
+
+.accordion-two.content.close {
+  max-height: 0;
+}
+
+.th-1 {
+  width: 120px;
+}
+
+.accordion-two.content.open {
+  max-height: 500px;
```

Comment:
I believe these styles: `.rotate90`, `.rotate180::before`, `.accordion-two.content`, `.accordion-two.content.close`, `.th-1`, `.accordion-two.content.open` applies **only** to the `Accordion` component you created, how about you scope them to the `Accordion` component by keeping it in a separate style file instead of keeping it global like this?

You can make refrence to a component like `SearchDrug` to see how the style modularization is done.

Reply:
done